### ISSUE TITLE
Add select list in "New" button

### ISSUE THREAD

**Issue by simozhan** [AllenFang/react-bootstrap-table#387]
Hi, I am trying to add inline select list to the table item, i find there is editable attribute which can be used, for example: 
        var editable = {
            type: "select",
            options:{
            values: ["choice1", "choice2","choice3"]
            }
        };
it works pretty good in the table, but when i click the New button, the input is still text, not select list.
so i am wondering if it is ok to make New button and the table has the same attribute to make this happen?

Thanks!

**Comment by AllenFang** [comment]
@simozhan, could you please show me a whole example, I think your configuration is well, maybe something wrong

**Comment by simozhan** [comment]
@AllenFang this is my snippet:
```
render: function () {
        var cellEditProp = {
            mode: "click",
            blurToSave: true,
            beforeSaveCell: this.onBeforeSaveCell
        };
        const options = {
            ignoreEditable: true
        };
        var row_editable = {
            type: "select",
            options:{
            values: this.props.roles
            }
        };
        var contact_editable = {
            type: "select",
            options:{
            values: this.props.contacts
            }
        };
        var contacts = this.props.data.map(function (c) {
	......
        });
        return (
                <div>
                    <Col md={8} mdOffset={2}>
                        <h3 className="text-center"> Results </h3>
                        <BootstrapTable data={contacts} striped={true} hover={true} condensed={true} insertRow={true} cellEdit={cellEditProp} options={options}>
                            <TableHeaderColumn dataField="id" dataAlign="center" width="80" isKey={true} autoValue={true} hidden={true}>ID</TableHeaderColumn>
                            <TableHeaderColumn dataField="component" dataAlign="center" editable={false} width="80">Component</TableHeaderColumn>
                            <TableHeaderColumn dataField="release" dataAlign="center" editable={false} width="60">Release</TableHeaderColumn>
                            <TableHeaderColumn dataField="contact" dataAlign="center" editable={contact_editable} width="180">Contact</TableHeaderColumn>
                            <TableHeaderColumn dataField="role" dataAlign="center" editable={row_editable} width="60">Contact Role</TableHeaderColumn>
                        </BootstrapTable>
                    </Col>
                </div>
        );
    }
```

Thanks

**Comment by AllenFang** [comment]
@simozhan, it's a bug caused by ```ignoreEditable```, I'll fix it in these day. Thanks your feedback :)

**Comment by AllenFang** [comment]
@simozhan, fixed it on ```v2.1.2```

**Comment by simozhan** [comment]
@AllenFang Thanks!
